src/main/java/time_manager_UI/TimeBlockAvailabilityUI.java

```diff
@@ -0,0 +1,74 @@
+package time_manager_UI;
+import javax.swing.*;
+import javax.swing.table.DefaultTableCellRenderer;
+import javax.swing.table.DefaultTableModel;
+import javax.swing.table.TableColumnModel;
+import UI.CentralWindow;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+
+/**
+ * A TimeBlockManagerUi that uses JFrame to show a window that illustrates the Availability of a User.
+ * Created:2022-11-14
+ * Last updated:2022-11-16
+ *
+ * @author Amir Bare
+ */
+public class TimeBlockAvailabilityUI extends CentralWindow {
+    private JPanel mainPanel;
+    private JTable showTable;
+    private JButton backbutton;
+    private JButton changeAvailabilityButton;
+
+    public TimeBlockAvailabilityUI() {
+        super();
+        this.setTitle("User Availability");
+        this.setContentPane(this.mainPanel);
+        this.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        this.pack();
+        this.centreWindow();
+        showTable.setEnabled(false);
+        createTable();
+
+        backbutton.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+            }
+        });
+
+        changeAvailabilityButton.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+            }
+        });
+    }
+
+    private void createTable(){
+        Object[][] contents = {
+                {"Monday"," From       to   "},
```

Comment:
I don't know how you intend on implementing what will actually give this UI its from: and to: times, but I'd definitely keep this implementation you made in mind in case you need to change it. Think of what your presenters are intended to do, and what information you can have them give.